contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -17,11 +17,11 @@ GRANT SELECT on sys.babelfish_sysdatabases TO PUBLIC;
 -- BABELFISH_SCHEMA_PERMISSIONS
 CREATE TABLE sys.babelfish_schema_permissions (
   dbid smallint NOT NULL,
-  schema_name NAME NOT NULL,
-  object_name NAME NOT NULL,
-  permission NAME NOT NULL,
-  grantee NAME NOT NULL,
-  object_type NAME,
+  schema_name NAME NOT NULL COLLATE "C",
+  object_name NAME NOT NULL COLLATE "C",
+  permission NAME NOT NULL COLLATE "C",
+  grantee NAME NOT NULL COLLATE "C",
+  object_type NAME COLLATE "C",
```

Comment:
As discussed offline, lets use sys.database_default here and ensure that we use correct collation for catalog scan/lookup (heap or index scan).